packages/test-react/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "test-react",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "test": "vitest --run",
+    "type-check": "tsc --noEmit"
+  },
+  "devDependencies": {
+    "@tanstack/react-query": "5.45.0",
+    "@testing-library/jest-dom": "6.4.6",
+    "@testing-library/react": "16.0.0",
+    "@trpc/client": "11.0.0-rc.403",
+    "@trpc/react-query": "11.0.0-rc.403",
+    "@trpc/server": "11.0.0-rc.403",
+    "@types/react": "18.3.3",
+    "@types/react-dom": "18.3.0",
+    "jsdom": "24.1.0",
```

Comment:
happy-dom is faster than jsdom, albeit it is lacking some of the web APIs, one should consider trying to see if it's adequate for this package

https://github.com/capricorn86/happy-dom